developer_workflow.md

```diff
@@ -0,0 +1,40 @@
+* Identify what needs to be done to complete this card. What does it mean for a feature to be complete? (create and reference separate post on this)
+* Create a local feature branch based off master. Give your branch a short descriptive name based on the card on which you are working.
+* Determine an appropriate level of testing for your branch, and decide how you will test your work. (create and reference separate post on testing)
+* When you are finished with your feature, make sure the entire test suite passes and your work hasn't broken the build.
+* Verify everything works locally with real data. Play with the feature and ensure the user experience is good.
+* Add to the README as you develop, making note of any steps required for deployment and testing: migrations, running scripts, updates to the server, etc. See our (readme document)[https://github.com/HaughtCodeworks/TeamGuide] for more on this.
+* Consider if you need to break your changes into multiple commits. Once you've grouped your changes into logical commits, add a good commit message. Make sure that you are only committing the changes that make sense for this feature.
+* Push your branch to origin, and avoid including sensitive files in source control.
+* Submit a pull request on Github/BitBucket, and tag relevant reviewers from your team.
+* Push your branch to the integration environment after verifying no one else on the team is using it. Make sure everything works in a true server environment.
+* Move your card on the planning board to the "In Review" column.
+
+### Pull Request Process
+
+* Post in team chat that your pull request is ready for review and your branch is on the integration environment.
+* A team member other than the author reviews the pull request. Follow the (Code Review guidelines)[https://github.com/HaughtCodeworks/TeamGuide] to avoid miscommunication. Make comments and ask questions directly on lines of code in the GitHub/ BitBucket repo or in team chat.
+* Reviewers verify integration functionality before approving a pull request. Verification in integration should happen with a (QA mindset)[https://github.com/HaughtCodeworks/TeamGuide].
+* When satisfied, the reviewer approves and comments on the pull request “Ready to merge" or "+1”.
+* The author of the PR rebases interactively. Squash commits like “Fix whitespace” into one or a small number of valuable commit(s). Edit commit messages to reveal intent. Run the test suite.
+* View a list of new commits and view changed files. Merge your branch into master.
+* Force push your branch, which will allow GitHub/ BitBucket to automatically close your PR once your commits are pushed to master.
+* Push to master.
+* Delete your remote feature branch.
+* Delete your local feature branch.
+
+### Staging your Changes
+
+* Once a pull request is approved and merged to master, master should be deployed to staging.
+* Perform any required steps listed in the README.
```

Comment:
This should be any required steps outlined in the card.  They may reference the README but since the README has all instructions for bootstrapping a project many of the steps would not be relevant.